src/App.js

```diff
@@ -1,65 +0,0 @@
-import React, { Component } from 'react';
-import logo from './lukelogo.png';
-import axios from 'axios'
-import './App.css';
-
-import Animaladd from './components/Animaladd'
-import Animal from './components/Animal'
-
-const expressURL = 'http://localhost:3032/api/animals'
-
-class App extends Component {
-  constructor() {
-    super()
-
-    this.state = {
-      frontArray: []
-    }
-  }
-
-  componentDidMount = () => {
-    axios.get(expressURL).then(res => {
-      this.setState({ frontArray: res.data })
-    }).catch(error => {
-      console.error(error)
-    })
-  }
-
-  addAnimal = (name, url) => {
-    axios.post(expressURL, {name: name, imageUrl: url}).then(res => {
```

Comment:
quick shorthand trick, if you named your parameter `imageUrl`, you could pass your object using ES6 shorthand like: 

```
axios.post(expressURL, { name, imageUrl })
```